test/unit/metrics/reporters/datadog.spec.js

```diff
@@ -212,14 +213,14 @@ describe("Test Datadog Reporter class", () => {
 			});
 			reporter.init(registry);
 
-			registry.register({ name: "os.datetime.utc", type: "gauge" }).set(123456);
-			registry.register({ name: "test.info", type: "info" }).set("Test Value");
+			registry.register({ name: "os.datetime.utc", type: 3 }).set(123456);
```

Comment:
The registry is common, don't change the type. You should make the Datadog specific conversion inside the Datadog reporter and not in the common classes.

Reply:
But this is the test for datadog reporter, why is this common? They use numbers now, not names.